### PR TITLE
[ODS-5216]Plumbing to support the v5.2 and v5.3 migrations

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: EdFi.Ods.Utilities.Migration/DatabaseReference/5.2/Ed-Fi Standard/Artifacts/MsSql/Data/Ods/0020-SchoolYears.sql [hunk 1]
```diff
@@ -0,0 +1,125 @@
+-- SPDX-License-Identifier: Apache-2.0
```
Comment (bptillman) [inline EdFi.Ods.Utilities.Migration/DatabaseReference/5.2/Ed-Fi Standard/Artifacts/MsSql/Data/Ods/0020-SchoolYears.sql]: Not a comment about this specific file but more the folder structure. In previous 4 versioned folders (3.3, 3.4, 5.0, and 5.1) the name `EdFi.Ods.Standard` was used for the folder, but for these changes it was called `Ed-Fi Standard`. For consistency should we rename it here and in the 5.3 directory to match the prior folder versions?

File: EdFi.Ods.Utilities.Migration.Tests/MsSql/MigrationTests/Latest/GrandBendFullUpgradeTests/GrandBendFullUpgradeTests.cs [hunk 2]
```diff
@@ -15,7 +15,7 @@ namespace EdFi.Ods.Utilities.Migration.Tests.MsSql.MigrationTests.Latest.GrandBe
     public class GrandBendFullUpgradeTests : GrandBendTests
     {
         protected override EdFiOdsVersion FromVersion => EdFiOdsVersion.V2UpTo24;
-        protected override EdFiOdsVersion ToVersion => SqlServerMigrationTestsGlobalSetup.OdsMigrationManagerResolver.GetLatestSupportedUpgradeVersion(FromVersion, DatabaseEngine.SQLServer);
+        protected override EdFiOdsVersion ToVersion => EdFiOdsVersion.V51;
```
Comment (bptillman) [inline EdFi.Ods.Utilities.Migration.Tests/MsSql/MigrationTests/Latest/GrandBendFullUpgradeTests/GrandBendFullUpgradeTests.cs:18]: Should we make a note in ODS-5212 as a reminder to make sure that this gets changed back to confirm that it works when migrating to the latest version?

**Review by bptillman — COMMENTED** [review]
Just a few comments I made. Let me know what you think.

File: EdFi.Ods.Utilities.Migration.Tests/MsSql/MigrationTests/Latest/GrandBendFullUpgradeTests/GrandBendFullUpgradeTests.cs (hunk 2 above)
Comment (semalaiappan) [reply EdFi.Ods.Utilities.Migration.Tests/MsSql/MigrationTests/Latest/GrandBendFullUpgradeTests/GrandBendFullUpgradeTests.cs:18]: I updated it in  ticket ODS-5212 ticket as comments https://tracker.ed-fi.org/browse/ODS-5212

File: EdFi.Ods.Utilities.Migration/DatabaseReference/5.2/Ed-Fi Standard/Artifacts/MsSql/Data/Ods/0020-SchoolYears.sql (hunk 1 above)
Comment (semalaiappan) [reply EdFi.Ods.Utilities.Migration/DatabaseReference/5.2/Ed-Fi Standard/Artifacts/MsSql/Data/Ods/0020-SchoolYears.sql]: yes updated to EdFi.Ods.Standard

**Review by bptillman — APPROVED** [review]
Changes look good for merging

**Pull request merged**